package.json

```diff
@@ -17,7 +17,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI= react-scripts build",
```

Comment:
This will set `process.env.CI` to `false` and will make the Netlify build pass.

Reply:
Thread in Netlify's Forum: https://community.netlify.com/t/build-command-just-started-failing-create-react-app/17188/2?u=alejandronanez